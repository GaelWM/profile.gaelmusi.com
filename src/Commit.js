import React from "react";

const Commit = props => {
  return (
    <li key={props.index} className="list-group-item list-group-item-action">
      {props.message}
    </li>
  );
};

export default Commit;
