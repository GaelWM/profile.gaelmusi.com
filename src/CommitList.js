import React, { Component } from "react";

const CommitList = props => {
  return <ol className="list-group">{props.commits}</ol>;
};

export default CommitList;
