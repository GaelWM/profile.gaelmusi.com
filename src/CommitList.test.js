import React from "react";
import CommitList from "./CommitList";
import Commit from "./Commit";
import renderer from "react-test-renderer";

test("CommitList renders properly ", () => {
  const message = "Add login form";
  const commit = <Commit key="1" message={message} />;
  const commits = [];
  commits.push(commit);

  const component = renderer.create(<CommitList commits={commits} />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
