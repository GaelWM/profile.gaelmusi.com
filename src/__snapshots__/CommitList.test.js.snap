// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommitList renders properly  1`] = `
<ol
  className="list-group"
>
  <li
    className="list-group-item list-group-item-action"
  >
    Add login form
  </li>
</ol>
`;
